src/main/java/com/epam/aidial/cfg/domain/service/KeyService.java

```diff
@@ -1,15 +1,20 @@
 package com.epam.aidial.cfg.domain.service;
 
-
 import com.epam.aidial.cfg.dao.jpa.KeyJpaRepository;
 import com.epam.aidial.cfg.dao.mapper.KeyEntityMapper;
 import com.epam.aidial.cfg.dao.model.KeyEntity;
+import com.epam.aidial.cfg.dao.model.RouteEntity;
```

Comment:
looks like RouteEntity&InterceptorRunner are unused imports. should be removed

Reply:
done